deployments/ocis-nats/README.md

```diff
@@ -0,0 +1,48 @@
+# oCIS with NATS deployment example
+
+## Introduction
+
+This example shows how to deploy NATS with NATS as message bus and store.
+It will deploy an oCIS instance and NATS, preconfigured to work together.
+
+***Note***: This example is not intended for production use. It is intended to get a working oCIS
+with NATS running in Kubernetes as quickly as possible. It is not hardened in any way.
+
+## Getting started
```

Comment:
```suggestion
# oCIS with NATS Deployment Example

## Introduction

This example shows how to deploy NATS with NATS as message bus and store.
It will deploy an oCIS instance and NATS, preconfigured to work together.

***Note***: This example is not intended for production use and not hardened in any way. It is intended as template to get a working oCIS instance running with NATS in Kubernetes as quickly as possible.

## Getting started
```